word_segmentation.py

```diff
@@ -205,140 +294,115 @@ def clean_line(line):
     return line
 
 
-def preprocess_Thai(demonstrate):
+################# functions for processing texts #################
+
+def preprocess_thai(verbose):
```

Comment:
A better design would be a function that returns a list of `Line`s based on BEST data, and a separate function that computes F1 score, grapheme cluster dictionary, etc., given the list of `Line`s.